src/base/net/dnsupdater.cpp

```diff
@@ -300,7 +300,7 @@ QUrl DNSUpdater::getRegistrationUrl(const int service)
     switch (service)
     {
     case DNS::DYNDNS:
-        return {"https://www.dyndns.com/account/services/hosts/add.html"};
+        return {"https://account.dyn.com/entrance/"};
```

Comment:
I would suggest you move this simple change to another PR so it will get approved easily.

Reply:
Revert it in this PR and open #14794